### PR TITLE
bugs & value set importing

### PR DESCRIPTION
**Pull request by grahamegrieve**

### CODE REVIEW

**Comment by grahamegrieve** [comment]
@markiantorno review please

File: org.hl7.fhir.convertors/src/main/java/org/hl7/fhir/convertors/misc/OIDBasedValueSetImporter.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    if ("http://www.nlm.nih.gov/research/umls/rxnorm".equals(url)) {
+      if (csver.length() == 8) {
+        return csver.substring(4,6)+csver.substring(6,8)+csver.substring(0,4);
+      } else {
+        return csver;        
+      }
+      
+    }
+    return csver;
+  }
```
Comment (markiantorno) [inline org.hl7.fhir.convertors/src/main/java/org/hl7/fhir/convertors/misc/OIDBasedValueSetImporter.java:49]: Is there any possibility to put these URLs in their own static consts that be accessed?

Comment (markiantorno) [reply org.hl7.fhir.convertors/src/main/java/org/hl7/fhir/convertors/misc/OIDBasedValueSetImporter.java:49]: Are they used in other parts of the code?

File: org.hl7.fhir.convertors/src/main/java/org/hl7/fhir/convertors/misc/VSACImporter.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import org.hl7.fhir.utilities.cache.FilesystemPackageCacheManager;
+import org.hl7.fhir.utilities.cache.NpmPackage;
+import org.hl7.fhir.utilities.cache.ToolsVersion;
+import org.hl7.fhir.utilities.xml.XMLUtil;
+import org.w3c.dom.Element;
+
+public class VSACImporter extends OIDBasedValueSetImporter {
+
+  public static void main(String[] args) throws FileNotFoundException, FHIRException, IOException, ParseException {
+//    new PhinVadsImporter().importValueSet(TextFile.fileToBytes("C:\\work\\org.hl7.fhir\\packages\\us.cdc.phinvads-source\\source\\PHVS_BirthDefectsLateralityatDiagnosis_HL7_V1.txt"));
```
Comment (markiantorno) [inline org.hl7.fhir.convertors/src/main/java/org/hl7/fhir/convertors/misc/VSACImporter.java:34]: Commented code.

**Pull request merged**